### PR TITLE
Add dead simple API

### PR DESCRIPTION
**Pull request by honzajavorek**
Příprava na https://github.com/pyvec/guide/issues/11#issuecomment-411534372

### CODE REVIEW

**Comment by honzajavorek** [comment]
```
Error: Filename extension of 'index.html' (type text/html) does
not match Content-Type: application/json
```

Hmmm 🤔

**Comment by honzajavorek** [comment]
Nechápu? Travis řve nesmysly. Oddebuguju jindy.

File: pyvecorg/views.py [hunk 1]
```diff
@@ -11,18 +11,15 @@
...
-        return redirect(url_for('index', lang='en'))
-    return redirect(url_for('index', lang='cs'))
-
-
 @app.route('/favicon.ico')
 def favicon():
     static = os.path.join(app.root_path, 'static')
-    return send_from_directory(static, 'favicon.ico',
-                               mimetype='image/vnd.microsoft.icon')
+    return send_from_directory(static, 'favicon.ico', mimetype='image/x-icon')
```
Comment (hroncok) [inline pyvecorg/views.py]: According to the Internet Assigned Numbers Authority (IANA), all .ico file falls under the MIME type [image/vnd.microsoft.icon](http://www.iana.org/assignments/media-types/image/vnd.microsoft.icon) which is what Travis tells you anyway.

Comment (honzajavorek) [reply pyvecorg/views.py]: Oh! I understood the error message the other way around 🤦‍♂️ Thanks for the help!

Comment (honzajavorek) [reply pyvecorg/views.py]: (my comment in 029dad2 being a proof 😄 )

**Comment by honzajavorek** [comment]
So locally I get following with the new setup:

```
Error: Filename extension of 'favicon.ico' (type image/x-icon) does not match Content-Type: image/vnd.microsoft.icon
```

The previous setup passed fine locally, but blew up on CI. So is this Linux vs macOS thing?!

**Comment by honzajavorek** [comment]
So now it passes on CI, but it probably isn't portable to macOS ¯\\\_(ツ)\_/¯ Not such an issue as it all works with `serve`, only `freeze` has the error and I don't need `freeze` locally.

Ready for a re-review.

**Comment by encukou** [comment]
It could also be a difference in Python or Flask versions. Elsa is quite picky here (for good reasons actually, but sometimes unfortunately things break.)

File: pyvecorg/views.py [hunk 2]
```diff
@@ -11,23 +11,37 @@
...
 def index(lang):
     context = select_language(data, lang)
     context['lang'] = lang
     context['now'] = datetime.now()
     return render_template('index.html', **context)
+
+
+@app.route('/api.json')
+def api_redirect():
+    return redirect(url_for('api', lang=detect_lang(request)))
```
Comment (encukou) [inline pyvecorg/views.py]: This doesn't actually work for a static page.
You're aware of that, right?

**Comment by hroncok** [comment]
Could you rebase the mime change there and back not to be here?

File: pyvecorg/views.py (hunk 2 above)
Comment (hroncok) [reply pyvecorg/views.py]: Good catch. :)

**Comment by honzajavorek** [comment]
@encukou @hroncok final blessing?

**Pull request merged**